pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/RealtimeKinesisIntegrationTest.java

```diff
@@ -84,6 +84,7 @@
 
 
 @LocalstackDockerProperties(services = {"kinesis"})
+@Test(enabled = false)
```

Comment:
This itself should disable the test?

Reply:
seems like no from the previous CI run

Reply:
commenting out the entire block is another option :-P